package med.voll.api.domain.exception;

public record DadosValidacaoException(String mensagemDeErro) {
}
